docs/guides/redis/clustering/overview.md

```diff
@@ -14,15 +14,28 @@ section_menu_id: guides
-
-So basically it is a group of multiple Redis nodes where data is **automatically sharded across multiple Redis nodes**. And it also provides **some degree of availability during partitions**, that is in practical terms the ability to continue the operations when some nodes fail or are not able to communicate. However, the cluster stops to operate in the event of larger failures (for example when the majority of masters are unavailable).
+Redis Cluster is a native sharding solution for Redis, which provides automatic partitioning of data across multiple Redis nodes.
+It enables horizontal scalability, allowing Redis to handle larger amounts of data by splitting data across multiple Redis instances.
+Each Redis instance in a Redis Cluster can operate as a master or slave, providing failover and redundancy for increased availability and reliability. 
+Redis Cluster uses a consistent hashing algorithm to distribute keys across nodes, ensuring that the data is evenly balanced and minimizing the 
+overhead of rebalancing when nodes are added or removed. Redis Cluster also provides a distributed system for managing and maintaining the cluster,
+ensuring that data is always consistent and available even in the event of node failures.
 
 So in practical terms, what do you get with Redis Cluster?
 
-- The ability to **automatically split your dataset among multiple nodes**.
-- The ability to **continue operations when a subset of the nodes are experiencing failures** or are unable to communicate with the rest of the cluster. See the [reference](https://redis.io/topics/cluster-tutorial#redis-cluster-101).
+- **Horizontal Scalability**: Redis Cluster allows Redis to handle larger amounts of data by splitting the data across multiple nodes, enabling linear scalability as more nodes are added to the cluster.
+
+- **High Availability**: Redis Cluster provides automatic failover and redundancy, ensuring that data is always available and that the cluster can continue operating even in the event of node failures.
+
+- **Load Balancing**: Redis Cluster distributes keys evenly across nodes, ensuring that the cluster is balanced and reducing the overhead of rebalancing when nodes are added or removed.
+
+- **Consistent Data**: Redis Cluster provides a distributed system for managing and maintaining the cluster, ensuring that data is always consistent and available.
+
+- **Easy Administration**: Redis Cluster provides a centralized management interface for the cluster, making it easier to manage and monitor large-scale Redis setups.
+
+- **Fast Performance**: Redis Cluster provides fast, in-memory data access, enabling fast and responsive applications.
 
+- **Simplified operations**: Redis Cluster eliminates the need for manual sharding, enabling a simpler and more automated way to scale Redis.
 ![redis-cluster](/docs/images/redis/redis-cluster.png)
 
 > Image reference [here](https://redislabs.com/redis-features/redis-cluster).
```

Comment:
this reference is redirecting to redis.com, not to an image.

Reply:
Removed Ref